GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -654,6 +698,25 @@ namespace gdjs {
       this._pausedSounds.length = 0;
     }
 
+    preloadAudioFile(
```

Comment:
Is there a reason for moving this to the public interface? We now have a risk users will call and rely on this function - which we probably don't want to?

Reply:
Probably a misunderstanding from my side. I didn't think making it outside would make it available to the users, and I felt it was cleaner this way. My bad!

Reply:
No worries! 
In not performance sensitive code it's better to keep things either inside the function or outside the whole class if it's a pure helper function - in both cases it's not accessible by the outside world (so it's even better than putting an underscore to say "this is private don't call me").